File: templates/javaee-app/model/src/main/java/org/sgodden/tom/model/CustomerOrderLine.java
package org.sgodden.tom.model;

/**
 * A line on a customer order.
 * 
 * @author Simon Godden
 */
@SuppressWarnings("serial")
public class CustomerOrderLine extends AbstractIdentity {

	private String packageType;
	private String descriptionOfGoods;

	public String getPackageType() {
		return packageType;
	}
	public void setPackageType(String packageType) {
		this.packageType = packageType;
	}
	public String getDescriptionOfGoods() {
		return descriptionOfGoods;
	}
	public void setDescriptionOfGoods(String descriptionOfGoods) {
		this.descriptionOfGoods = descriptionOfGoods;
	}

}

File: templates/scala-mongodb/model/src/main/scala/org/sgodden/tom/model/CustomerOrderLine.scala
package org.sgodden.tom.model

import com.novus.salat.annotations.raw.Persist

case class CustomerOrderLine() {
  @Persist
  private var packageType: String = null
  @Persist
  private var descriptionOfGoods: String = null

  def getPackageType = packageType
  def setPackageType(packageType: String) { this.packageType = packageType }

  def getDescriptionOfGoods = descriptionOfGoods
  def setDescriptionOfGoods(descriptionOfGoods: String) = this.descriptionOfGoods = descriptionOfGoods
}
